lib/smooth/include/smooth/core/filesystem/MMCSDCard.h

```diff
@@ -18,6 +18,11 @@ limitations under the License.
 #pragma once
 
 #include "SDCard.h"
+
+#ifndef SOC_SDMMC_HOST_SUPPORTED
+#error "Your chip does not support SDMCC_HOST. See https://github.com/PerMalmberg/Smooth/issues/171"
```

Comment:
Perhaps reference some applicable IDF documentation instead?

Reply:
That's a good idea, but I'm not sure how to implement it.
`SOC_SDMMC_HOST_SUPPORTED` is described on chip-specific pages, e.g.
```
https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/system/soc_caps.html
https://docs.espressif.com/projects/esp-idf/en/latest/esp32c3/api-reference/system/soc_caps.html
```

Correct URL can be computed by inserting `CONFIG_IDF_TARGET` into the error message, but I don't know how to do that inside the `#error`.